docs/command/commands.md

```diff
@@ -0,0 +1,84 @@
+---
+id: commands
+title: C3PM commands
+---
+### How to use c3pm commands?
+
+`ctpm add`
+
+This commands adds a new dependency to the current project. It downloads, unpacks, builds and installs the dependency following the dependency handling spec.
+
+`packages` is a list of terms, each indicating a package to add as a dependency. Each entry should follow the format:
+
+```bash
+name[@version]
+```
+
+Where `name` is the name of the package to install, and `version` describes the version to install, defaulting to `latest`.
+
+###### Options
+
+* `--force` - Will redo the whole installation process even if the package is already installed.
+
+### `ctpm init`
```

Comment:
maybe `ctpm init <?path>` to explicit the fact that it can take a path as parameter ?